### DISCUSSION TITLE
"required without" functionality

### DISCUSSION THREAD

**Discussion by jpetryk** [directus/directus#4513] [Feature Requests]
Has there been any discussion or thought put toward "required without" functionality? Essentially, one particular field must exist if one or more other fields don't exist? It would allow someone to basically create a group of 2 or more fields for which at least one must be filled out for the the form to validate, such as "You must fill out one of title, subtitle or description".

**Comment by rijkvanzanten** [comment]
I believe that would be possible with this idea: https://github.com/directus/directus/discussions/2993 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
